frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts

```diff
@@ -10,3 +11,9 @@ export const getVMIApiPath = (vmi: VMIKind) =>
   `${VirtualMachineInstanceModel.apiVersion}/namespaces/${getNamespace(vmi)}/${
     VirtualMachineInstanceModel.plural
   }/${getName(vmi)}`;
+
+const getVMIStatusConditions = (vmi: VMIKind) =>
+  _.get(vmi, 'status.conditions', []) as VMIKind['status']['conditions'];
```

Comment:
For all new selectors, we should use direct access instead of the lodash. Sort of:
```
vmi && vmi.status && vmi.status.conditions || []
```

Reply:
I was able to remove the function entirely by using an existing function in `selectors/vmi/basic.ts`.